### DISCUSSION TITLE
Installation instructions?

### DISCUSSION THREAD

**Discussion by VashJuan** [curl/curl-for-win#73] [Ideas]
Windows forks are most likely to install via a package manager or setup executable. 

Many Windows users will be unsure where to put curl.exe from the zip file: app/roaming/or Windows/system 32, etc...

It would be nice to have instructions or a pointer to same in the readme or downloads page.

Thx!

**Comment by vszakats** [comment]
I think the point is that there is no installation. Where to unpack is up the everyone's use case, preference, or permissions. Another optional step is adding that location to `PATH`.

For those uncertain doing these manually, it's probably easiest to rely on winget, scoop or Chocolatey, which make these steps automatic and use their own locations to make it work as a one-step process. These package managers all use curl-for-win binaries under the hood.

That said, I'd be happy linking to a document (a [curl wiki](https://github.com/curl/curl/wiki) page?) that's community maintained.

**Comment by VashJuan** [comment]
I know a bit more after researching further!

I think it would help to add a sentence stating that "Windows users should generally wait for Microsoft to update curl via standard Windows Updates". The Curl executable is a trusted system file and not easy to replace or update.

More info at:
https://daniel.haxx.se/blog/2023/04/24/deleting-system32curl-exe/

**Comment by vszakats** [comment]
Yes, overwriting the base OS, including its bundled curl.exe with any other curl is strongly not recommended. Unzipping the archives distributed here will never overwrite those.

The Windows bundled curl is mentioned on the binary download page: <https://curl.se/windows>.

**Reply by VashJuan** [reply]
It still might be nice to mention - on the downloads page - that "it is not generally advised to manually update Curl. That's handled by the packages/operating system it is delivered with." (If this wording is accurate!)

**Reply by vszakats** [reply]
This depends on _which_ curl somebody means to update. And that there is the question of identifying it.

I still feel that protecting against the OS files being messed up is better addressed by Microsoft, rather than each person / project who happens to build an independent `curl.exe`. For such independent curl binaries, we actually recommend to keep them updated, either manually or via a package manager mentioned above.
